### PR TITLE
Update mongo documentation so that it doesn't depend on having an admin collection.

### PR DESCRIPTION
**Pull request by Christian-Nunnally**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/systemlink-operations-handbook/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Update the remote mongo configuration steps to work regardless of whether an admin collection exists.

### Why should this Pull Request be merged?

Remote mongo databases might not have a default admin collection, but it's not necessary if you don't include a connection name in the custom connection string field.

### What testing has been done?

Manually verified the above steps allow services to connect to the newly configured remote DB.

### CODE REVIEW

**Review by cameronwaterman — APPROVED** [review]

File: handbook/data-stores/mongodb.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ If you have not setup an Atlas cluster before, refer to [Getting started with At
     This will need to be updated to 
     
     ```url
-    mongodb+srv://<username>:<password>@<cluster>/admin?retryWrites=true&w=majority
+    mongodb+srv://<username>:<password>@<cluster>
```
Comment (JD-Robertson) [inline handbook/data-stores/mongodb.md:127]: Why did you remove the options?
